internal/util/vault_tokens.go

```diff
@@ -63,7 +63,7 @@ type standardVault struct {
 	VaultClientCert    string `json:"VAULT_CLIENT_CERT"`
 	VaultClientKey     string `json:"VAULT_CLIENT_KEY"`
 	VaultNamespace     string `json:"VAULT_NAMESPACE"`
-	VaultSkipVerify    *bool  `json:"VAULT_SKIP_VERIFY"`
+	VaultSkipVerify    string `json:"VAULT_SKIP_VERIFY"`
```

Comment:
@nixpanic is this member not changed to `VaultCAVerify` from `VaultSkipVerify` because of backward compatibility ?

Reply:
No, we use `VaultSkipVerify` here, as that matches the `VAULT_SKIP_VERIFY` environment name Vault uses. The `VaultCAVerify` name is used the in the `vaultTokenConf` struct as that one matches the configuration options Ceph-CSI uses.

Reply:
Sure.. lets keep as it is then